2020-02-Spring/0_arrays_matrix/README.md

```diff
@@ -141,21 +153,122 @@ Go to [Top](#top)
 <!-- Don't remove -->
 <a name="s3"/>
 
-### 3. SOLUTION 3 TODO :bug:
-
-Source: TODO :bug:
-
-#### Naive/Simple Solution 
-
-TODO :bug:
-
-#### Optimal Solution
-
-TODO :bug:
+### 3. Word Search Solution 
+
+#### Solution
+
+
+Time Complexity: O(N * 4^{L}) where N is the number of cells in the board and L is the length of the word to be matched.
+
+
+##### Java
+```java
```

Comment:
Having both solutions in the Markdown file is perfectly fine, but it might be overkill for some folks. Completely up to you to decide whether to keep them both or remove one of them.